src/imitation/scripts/config/parallel.py

```diff
@@ -52,10 +53,33 @@ def debug_log_root():
   search_space = {"config_updates": {"log_root": "/tmp/debug_parallel_ex"}}
 
 
+@parallel_ex.named_config
+def _generate_test_data():
```

Comment:
Why is this prefixed with `_`? This isn't a usual named config to call, but it's not a module-private function, so it feels a little misleading.

Reply:
Removing `_`.